### ISSUE TITLE
Add trinity EMS

### ISSUE THREAD

**Issue by livgust** [livgust/covid-vaccine-scrapers#15]
HARD. Requires paging through a calendar view and clicking on dates to see slots available. Didn't notice any helpful routes returning JSON or anything.

**Comment by pfurbacher** [comment]
Can you assign it to me? I have much of it written -- stepping through each month in the calendar, evaluating days for appointments. What remains is verifying the selectors for days with appointments, parsing the number from XHR data for such days, and posting to s3 when calendar day attributes change to possibly reflect availability.

**Comment by livgust** [comment]
Hi @pfurbacher , do you have an update on this?

**Comment by pfurbacher** [comment]
I started refactoring to make the code more testable, but if you'd like to get something into the repository to look at, I can clean the existing code up (e.g., remove log statements?) and then request guidance on how to do the pull request. I'm not well-versed in Git, having mostly with older VCS packages.

**Comment by kriation** [comment]
The Trinity EMS tests that ran as part of an unrelated PR (#176) earlier this morning are failing.
```
  1) getTotalNumberOfAppointments
       works for single result with date availability:

      AssertionError: expected 0 to equal 8
      + expected - actual

      -0
      +8
      
      at Context.<anonymous> (test/metrics.js:24:17)
      at processImmediate (internal/timers.js:461:21)

  2) getTotalNumberOfAppointments
       works for array with date availability:

      AssertionError: expected 0 to equal 38
      + expected - actual

      -0
      +38
      
      at Context.<anonymous> (test/metrics.js:43:17)
      at processImmediate (internal/timers.js:461:21)
```

**Comment by tkjones117** [comment]
@kriation what makes you say that these are Trinity EMS Tests? these are in `metrics.js`, I think unrelated to Trinity EMS

**Comment by kriation** [comment]
@tkjones117 My apologies! I didn't note the files referenced in the stack trace. Disregard. :)

**Comment by tkjones117** [comment]
no problem! thanks for reporting it

**Comment by livgust** [comment]
fixed! sorry about that

**Comment by livgust** [comment]
Closed by #137
